item-spec/item-spec.md

```diff
@@ -106,6 +108,20 @@ by multiple files - all should be linked to. It is generally recommended that di
 levels or formats are not exhaustively listed in an Item, but instead are represented by related
 Items that are linked to, but the best practices around this are still emerging.
 
+#### geometry
+
+Defines the full footprint of the asset represented by this item, formatted according to RFC 7946.
+
+The footprint should be the default GeoJSON geometry, though additional geometries can be included.
```

Comment:
I don't quite understand what this sentence is communicating. What is a "default GeoJSON geometry"? Are there non-default GeoJSON geometries?

Reply:
I'm not sure either. It was there before this PR.

Reply:
If we don't understand it, we should remove it, ya?

Reply:
It's there since the beginning, it seems. I guess the "default" is meant to be some kind of a counterpart to the "additional geometries". I'm actually not so sure how additional geometries could be added?! Maybe we should remove the whole paragraph?